nef/Utils/Bookmark.swift

```diff
@@ -0,0 +1,54 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+
+@propertyWrapper
+struct Bookmark {
+    private let key: String
+    private let storage = UserDefaults.standard
```

Comment:
Why not adding this as an init parameter and set its default there? It can be more versatile.

Reply:
sure!